lomond/errors.py

```diff
@@ -1,31 +1,31 @@
 from __future__ import unicode_literals
 
 
-class Error(Exception):
+class WebSocketError(Exception):
```

Comment:
This is to allow devs to import the root exception with a more descriptive name.